CHANGELOG.md

```diff
@@ -2,7 +2,7 @@
 
 This is a major release
 
-* [TK-369](https://tickets.puppetlabs.com/browse/TK-369) Move Jetty dependency to 9.4.1
+* [TK-369](https://tickets.puppetlabs.com/browse/TK-369) [TK-440](https://tickets.puppetlabs.com/browse/TK-440) Move Jetty dependency to 9.4.4.
```

Comment:
That's intentional as we had multiple tickets for bumping the Jetty version. I can just pick one if you prefer.